Resources/Meta Analysis/description.json

```diff
@@ -1,8 +1,8 @@
 {
 	"moduleDescription" :
 	{
-		"title" : 		"Meta Analysis",
-		"description" : "Meta Analysis module.",
+		"title" : 		"Meta-Analysis",
```

Comment:
You can add "name" here and let that be "Meta Analysis". That solves the problem you have now. Otherwise rename the folder.

If name isn't entered title is used.

Reply:
Ah... The Dutch description file has a name, but the English one does not.
Thanks for having a look!

@sophieberkhout can you add `name: "Meta Analysis"` to the moduleDescription of this file?

Reply:
Done!